packages/flutter/test/material/scaffold_test.dart

```diff
@@ -552,7 +552,7 @@ void main() {
               right: 50.0,
               bottom: 60.0,
             ),
-            viewInsets: const EdgeInsets.only(bottom: 70.0),
+            viewInsets: const EdgeInsets.only(bottom: 200.0),
```

Comment:
Would test one more time that content is moved by just the persistent footer and bottom tabs if the test doesn't exist already

Reply:
Added a test.